src/static/helpers/slasHelper.ts

```diff
@@ -141,22 +152,25 @@ export async function authorize(
     redirect: isBrowser ? 'follow' : 'manual',
   };
 
-  const options = {
+  const {hint, redirectURI, usid, ...restOfParams} = parameters;
+  const opts = {
     parameters: {
       client_id: slasClient.clientConfig.parameters.clientId,
       channel_id: slasClient.clientConfig.parameters.siteId,
       ...(clientOptions.codeChallenge && {
         code_challenge: clientOptions.codeChallenge,
       }),
-      ...(parameters.hint && {hint: parameters.hint}),
+      ...(hint && {hint}),
       organizationId: slasClient.clientConfig.parameters.organizationId,
-      redirect_uri: parameters.redirectURI,
+      redirect_uri: redirectURI,
       response_type: 'code',
-      ...(parameters.usid && {usid: parameters.usid}),
+      ...(usid && {usid}),
+      ...restOfParams,
```

Comment:
Is there a use case where `...restOfParams` should be last instead of first? In my opinion, `..restOfParams` should go on top before we start defining the other parameter properties. Imagine a developer that uses javascript instead of typescript and so they're able to define `codeChallenge: '<insert-custom-string>` in the parameters object they pass in. That would break implementation and so we probably don't want to allow the ability to override these properties already set by us, which is a breaking change